engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -460,9 +461,14 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       val updatedData = startRunnableCalls(data)
       self ! CheckForWorkflowComplete
       stay() using updatedData
-    case Event(CallStarted(callKey), data) if !data.isPending(callKey) =>
+    case Event(CallStarted(callKey, maybeCallLogs), data) if !data.isPending(callKey) =>
       executionStore += callKey -> ExecutionStatus.Running
-      persistStatus(callKey, ExecutionStatus.Running)
+      // TODO: We're not using/waiting for the future results
+      // TODO: This should be in a single transaction!
```

Comment:
Yeah too bad our DB API doesn't return `DBIO[A]`s that could be sequenced and wrapped in a transaction